nb2workflow/nbadapter.py

```diff
@@ -124,7 +124,18 @@ def parse_nbline(line: str, nb_uri=None) -> Optional[dict]:
 
 
 def owl_type_for_python_type(python_type: type):
-    return "http://www.w3.org/2001/XMLSchema#"+ python_type.__name__ 
+    out_type = python_type.__name__
+
+    if python_type == int:
+        out_type = 'integer'
+
+    if python_type == str:
+        out_type = 'string'
+
+    if python_type == bool:
+        out_type = 'boolean'
+
+    return "http://www.w3.org/2001/XMLSchema#" + out_type
```

Comment:
what was there was a simple enough placeholder.

Now it's better, but it still creates cases when references in non-existent URIs will be created.
I guess we need to defer to our own namespace (odahub...) in the cases when python type does not correspond to anything in XMLSchema

Reply:
is the existence of a certain python type within the XML schema verified anywhere else?

Reply:
When we publish out metadata it will be recognized as incomplete. It's best not to create incomplete metadata since are onto it here.

There are no python types in XML schema, there is just a correspondence to it. We can express here our best interpretation of the correspondence without knowingly creating non-existent URIs.

Reply:
now it uses odahub namespace in case the python_type passed as parameter is not found the correspondent typew in the XML schema